src/foam/nanos/app/SupportConfig.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * @license
+ * Copyright 2020 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.nanos.app',
+  name: 'SupportConfig',
+
+  documentation: 'Support configuration for email',
+
+  properties: [
+    {
+      class: 'foam.core.FObjectProperty',
```

Comment:
@lchanmann why are we doing this? short term?

Reply:
Theme has supportConfig and supportConfig has emailConfig, if set the email from, display name and replyTo headers will be coming from the theme, if not set it will get the email config from "emailConfig" service.